src/serialization/base64.cpp

```diff
@@ -0,0 +1,90 @@
+#include "serialization/base64.hpp"
+
+#include <vector>
+
+namespace {
+const std::vector<int>& decode_map(){
```

Comment:
Nitpicking: opening braces of functions and namespaces belong to their own lines. (Same in lines 16 and 21.)